web/src/bootstrap_typeahead.ts

```diff
@@ -468,9 +484,9 @@ class Typeahead<ItemType extends string | object> {
         this.$container.remove();
         const events = ["blur", "keydown", "keyup", "keypress", "mousemove"];
         for (const event_ of events) {
-            this.$element.off(event_);
+            $(this.input_element.$element).off(event_);
```

Comment:
The `$()` wrapping here seems surprising; `$element` is already a JQuery object.

Reply:
Yeah it seemed like I needed to take the two possible types of `$element` and collapse it into a shared type.

This is the error I got, open to other suggestions that are better:

```
No overload matches this call.
  Overload 1 of 2, '(events: TypeEventHandlers<HTMLElement, any, any, any>, selector?: string | undefined): JQuery<HTMLElement>', gave the following error.
    Argument of type 'string' is not assignable to parameter of type 'TypeEventHandlers<HTMLElement, any, any, any>'.
  Overload 2 of 2, '(event?: TriggeredEvent<HTMLInputElement, any, any, any> | undefined): JQuery<HTMLElement>', gave the following error.
    Argument of type 'string' is not assignable to parameter of type 'TriggeredEvent<HTMLInputElement, any, any, any>'.ts(2769)
const event_: string
```